packages/jasmine-parameterized/README.md

```diff
@@ -0,0 +1,65 @@
+# Jasmine Parameterized
+
+The `@ni/jasmine-parameterized` library provides utility functions for writing jasmine parameterized tests.
```

Comment:
nit: just a bit more context in case someone stumbles on this package

```suggestion
The `@ni/jasmine-parameterized` library provides utility functions for writing [Jasmine](https://jasmine.github.io/) parameterized tests.
```